src/lib/spinel/radio_spinel_impl.hpp

```diff
@@ -1445,30 +1461,37 @@ void RadioSpinel<InterfaceType, ProcessContextType>::HandleTransmitDone(uint32_t
     aBuffer += unpacked;
     aLength -= static_cast<uint16_t>(unpacked);
 
+    unpacked = spinel_datatype_unpack(aBuffer, aLength, SPINEL_DATATYPE_BOOL_S, &framePending);
+    OT_UNUSED_VARIABLE(framePending);
```

Comment:
nit: move to the top of the function?

[STYLE_GUILD.md](https://github.com/openthread/openthread/blob/master/STYLE_GUIDE.md)
> All unused parameters shall be declared as such using the OT_UNUSED_VARIABLE macro at the top of a function or method before all local variable declarations.